Symfony/CS/Fixer/Symfony/PhpdocToCommentFixer.php

```diff
@@ -139,7 +139,7 @@ private function isValidControl(Tokens $tokens, Token $docsToken, $controlIndex)
         $endIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
         $docsContent = $docsToken->getContent();
 
-        for ($index = $index + 1; $index < $endIndex; ++$index) {
+        for (++$index; $index < $endIndex; ++$index) {
```

Comment:
`$index = $index + 1` is 3 operations:
- read
- sum
- strore

When `++$index` is one.